roles/k3s/master/tasks/main.yml

```diff
@@ -160,6 +160,13 @@
       --kubeconfig ~{{ ansible_user }}/.kube/config
   changed_when: true
 
+- name: Fetch cluster config file from first master to push to agent nodes - allowing agent nodes to run k3s kubectl xxx (patching, convenience)
+  ansible.builtin.fetch:
+    src: ~{{ ansible_user }}/.kube/config
+    flat: true
+    dest: /tmp/kube_config.{{ apiserver_endpoint }}
```

Comment:
This makes the kubecfg readable for all users on the host. Can we find a way that limits read access to the current user?